### PR TITLE
cards: added icons tests cases, and more specification for titles

### PR DESCRIPTION
**Pull request by davifantasia**

### CODE REVIEW

**Comment by davifantasia** [comment]
@nkuehn I can't find the link of your comment on making cards more accessible, please include it here.

**Comment by emmenko** [comment]
Can you clarify why you chose to pass icons as strings? I thought we wanted to go for components.

```
icon={<MyIcon />}
```

**Comment by davifantasia** [comment]
> Can you clarify why you chose to pass icons as strings? I thought we wanted to go for components.
> 
> ```
> icon={<MyIcon />}
> ```

Fixed, thanks.

**Comment by emmenko** [comment]
Ok.

However, this will fail because those components do not exist yet, that's why the icons examples have been removed 😅

**Comment by davifantasia** [comment]
> Ok.
> 
> However, this will fail because those components do not exist yet, that's why the icons examples have been removed 😅

It builds according to Vercel :)

**Comment by emmenko** [comment]
Yes, cause the icon is not rendered as the card component hasn’t been implemented yet.

Anyway, just wanted to make you aware of this

**Comment by davifantasia** [comment]
> Anyway, just wanted to make you aware of this

Thanks. Please approve if you're good with the PR.

File: websites/docs-smoke-test/src/content/components/cards.mdx [hunk 1]
```diff
@@ -172,3 +173,53 @@ would | work   |
 ## Output
```
Comment (emmenko) [inline websites/docs-smoke-test/src/content/components/cards.mdx]: I think we forgot this last time

```suggestion
**Output**
```

**Pull request merged**